addons/account/tests/test_account_bank_statement.py

```diff
@@ -1416,14 +1416,14 @@ def test_statement_attachments(self):
 
         statement = self.env['account.bank.statement'].create({
             'name': 'test_statement',
-            'attachment_ids': [Command.set(attachment.ids)],
+            'linked_attachment_ids': [Command.set(attachment.ids)],
```

Comment:
@william-andre I saw that it works on your branch, i suppose i need that commit ? Or i'm a doing a dumb mistake ahah 
https://github.com/odoo/odoo/pull/117074/commits/eddea8a23e6bc379b5b6789db29340db62ab36de

Since it's a more framework change master would be better right ?